packages/underscore/package.js

```diff
@@ -1,14 +1,14 @@
 
 Package.describe({
   summary: "Collection of small helpers: _.map, _.each, ...",
-  version: '1.0.13',
+  version: '1.6.0',
```

Comment:
I do not feel so good doing this huge bump since we use it everywhere.

Maybe we could drop this along the release with DDP resumption 2.15

Reply:
For 2.15 I was hoping to jump to the original plan of 1.9, I did this 1.6 as that one is the latest closest to what we have been using (and I kept in the changes that were made by MDG, after this version there doesn't seem to be a need for them anymore, but the tests disagree so I will have to look more into it).